daemon/api/dependencies.py

```diff
@@ -1,26 +1,24 @@
 import os
-from pathlib import Path
-from http import HTTPStatus
 from contextlib import ExitStack
+from http import HTTPStatus
+from pathlib import Path
 from typing import Dict, List, Optional
 
+from fastapi import HTTPException, UploadFile, File, Query, Depends
 from pydantic import FilePath
 from pydantic.errors import PathNotAFileError
-from fastapi import HTTPException, UploadFile, File, Query, Depends
 
-from jina import Flow, __docker_host__
-from jina.helper import cached_property
-from jina.peapods.peas.helper import update_runtime_cls
-from jina.peapods.peas.container_helper import get_gpu_device_requests
+from jina import Flow
 from jina.enums import (
-    PeaRoleType,
-    SocketType,
     RemoteWorkspaceState,
 )
+from jina.helper import cached_property
```

Comment:
change to relative dependencies

Reply:
I think thats not possible here as its not in the jina folder but in the daemon folder?